Plugins/BuildServerIntegration/AppVeyorIntegration/AppVeyorAdapter.cs

```diff
@@ -189,6 +189,24 @@ private class Project
             public string QueryUrl;
         }
 
+        private string BuildPullRequetUrl(string repositoryType, string repositoryName, string pullRequestId)
+        {
+            switch (repositoryType)
```

Comment:
```suggestion
            switch (repositoryType.ToLowerInvariant())
```
(though `ToUpperInvariant` is faster)